ownCloud/Tasks/ScheduledTaskManager.swift

```diff
@@ -184,9 +184,12 @@ class ScheduledTaskManager : NSObject {
 
 		// Add requirements
 		var requirements = [String : Bool]()
-		if self.wifiDetected {
-			requirements[ScheduledTaskAction.FeatureKeys.runOnWifi] = true
-		}
+        var preferences = [String : Bool]()
```

Comment:
Are the changes in `ScheduledTaskManager.swift` maybe a left-over from a previous/different approach/branch? These changes don't seem to be used/needed by the rest of the PR.